### PR TITLE
add ghproxy grafana dashboard

### PR DESCRIPTION
**Pull request by droslean**

### CODE REVIEW

File: cluster/ci/config/prow/plugins.yaml [hunk 1]
```diff
@@ -943,6 +943,9 @@ config_updater:
     cluster/ci/monitoring/sinker.json:
       name: grafana-dashboard-sinker
       namespace: prow-monitoring
+    cluster/ci/monitoring/ghproxy.json:
```
Comment (hongkailiu) [inline cluster/ci/config/prow/plugins.yaml]: According to my experience last time, the change on `plugins.yaml` has to be merged as a separate PR before the rest.

File: cluster/ci/monitoring/Makefile [hunk 2]
```diff
@@ -5,6 +5,7 @@ prow-monitoring-deploy:
 	oc apply -f ./prometheus_expose.yaml
 	oc apply -f ./prometheus_operator_deploy.yaml
 	oc apply -f ./sinker_service_monitor.yaml
+	oc apply -f ./ghproxy_service_monitor.yaml
```
Comment (hongkailiu) [inline cluster/ci/monitoring/Makefile]: No need of `ServiceMonitor` for ghproxy unless it has its own /metrics endpoint.
if it only push to pushgateway like tide, i have done it already.

File: cluster/ci/monitoring/ghproxy.json [hunk 3]
```diff
@@ -0,0 +1,801 @@
+{
```
Comment (hongkailiu) [inline cluster/ci/monitoring/ghproxy.json]: NIP for now: Give it a try with mixins and jsonnet?

File: cluster/ci/monitoring/Makefile (hunk 2 above)
Comment (droslean) [reply cluster/ci/monitoring/Makefile]: addressed

File: cluster/ci/monitoring/ghproxy.json (hunk 3 above)
Comment (droslean) [reply cluster/ci/monitoring/ghproxy.json]: can be done in follow-up?

**Comment by droslean** [comment]
> Have you fixed the words being cut off in the panel Cache Efficiency?
Yep! [ghproxy.json#L180](https://github.com/openshift/release/blob/3e27fee712e7655068f8ebc40055fec1114c07c1/cluster/ci/monitoring/ghproxy.json#L180)

File: cluster/ci/config/prow/plugins.yaml (hunk 1 above)
Comment (droslean) [reply cluster/ci/config/prow/plugins.yaml]: it should be ok. What was the problem?

Comment (hongkailiu) [reply cluster/ci/config/prow/plugins.yaml]: Not sure. I did not try otherwise. :)

File: cluster/ci/monitoring/ghproxy.json (hunk 3 above)
Comment (hongkailiu) [reply cluster/ci/monitoring/ghproxy.json]: Sure.
